example/acroname_power_supply_example.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2014 Google Inc. All Rights Reserved.
```

Comment:
We should either rename 'example' to 'examples' (and maybe create subdirs for the various examples), or put examples in with their corresponding plugs. Alternatively, just mock up the example code in the docstring of the plug rather than having an actual full test script for it.